core/modules/wpublish/QueryChunkStatistics.h

```diff
@@ -0,0 +1,170 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2013-2016 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_QSERV_WSCHED_QUERYCHUNKSTATISTICS_H
+#define LSST_QSERV_WSCHED_QUERYCHUNKSTATISTICS_H
+
+// System headers
+
+// Qserv headers
+#include "wbase/Task.h"
+
+namespace lsst {
+namespace qserv {
+namespace wpublish {
+
+
+class QueryChunkStatistics;
+
+
+/// Statistics for a single user query.
+class QueryStatistics {
+public:
+    using Ptr = std::shared_ptr<QueryStatistics>;
+
+    explicit QueryStatistics(QueryId const& queryId) : _queryId{_queryId} {}
```

Comment:
Hmm, interesting, I'd at least expect one for unused parameter. We probably have suppressed that sort of warnings for some reason.